tools/exec.js

```diff
@@ -8,36 +8,28 @@ let executionOptions = {
 };
 
 function logWithPrefix(prefix, message) {
-  let formattedMessage = message.trim().split('\n')
-    .reduce((acc, line) => `${acc}${ acc !== '' ? '\n' : '' }${prefix} ${line}`, '');
-
-  console.log(formattedMessage);
+  console.log(
+    message.toString().trim()
+    .split('\n')
+    .map((line) => `${prefix.grey} ${line}`)
+    .join('\n')
+  );
 }
 
 function execWrapper(command, options = {}) {
   let proc = exec(command, options);
-  let title = options.title || command;
-  let log = message => logWithPrefix(`[${title}]`.grey, message);
-
-  if (executionOptions.verbose) {
-    let output = (data, type) => {
-      logWithPrefix(`[${title}] ${type}:`.grey, data.toString());
-    };
-    proc = proc.progress(({stdout, stderr}) => {
-      stdout.on('data', data => output(data, 'stdout'));
-      stderr.on('data', data => output(data, 'stderr'));
-    })
-    .then(result => {
-      log('Complete'.cyan);
-      return result;
```

Comment:
Since this function is a wrapper around the `exec` function and it returns result, it would still be safer to return no? For example without returning it with the verbose flag this is what I get:

```
➜  react-bootstrap git:(pr851) ✗ ./tools/release patch --dry-run --verbose
DRY RUN
[git diff-index --name-only HEAD --] stdout: CHANGELOG.md
[git diff-index --name-only HEAD --] stdout: package.json
[git diff-index --name-only HEAD --] Complete
TypeError: Cannot read property 'stdout' of undefined
    at /Users/smithm/dev/react-bootstrap/tools/release-scripts/pre-conditions.js:7:17
    at _fulfilled (/Users/smithm/dev/react-bootstrap/node_modules/child-process-promise/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/smithm/dev/react-bootstrap/node_modules/child-process-promise/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/smithm/dev/react-bootstrap/node_modules/child-process-promise/node_modules/q/q.js:796:13)
    at /Users/smithm/dev/react-bootstrap/node_modules/child-process-promise/node_modules/q/q.js:604:44
    at runSingle (/Users/smithm/dev/react-bootstrap/node_modules/child-process-promise/node_modules/q/q.js:137:13)
    at flush (/Users/smithm/dev/react-bootstrap/node_modules/child-process-promise/node_modules/q/q.js:125:13)
    at doNTCallback0 (node.js:408:9)
    at process._tickCallback (node.js:337:13)
```